addOns/network/src/main/java/org/zaproxy/addon/network/ExtensionNetwork.java

```diff
@@ -138,6 +156,9 @@ public class ExtensionNetwork extends ExtensionAdaptor implements CommandLineLis
 
     private static final String I18N_PREFIX = "network";
 
+    private static final ImageIcon RATE_LIMIT_ICON =
```

Comment:
This should only be done if there's view.

Reply:
There is a view.

Reply:
Not when running as daemon or command line.